lib/beaker/host/pswindows.rb

```diff
@@ -23,5 +23,17 @@ def platform_defaults
       })
     end
 
+    attr_reader :scp_separator, :external_copy_base, :system_temp_path
+    def initialize name, host_hash, options
+      super
+
+      @scp_separator = '/'
+      # %TEMP% == C:\Users\ADMINI~1\AppData\Local\Temp
+      # is a user temp path, not the system path.  Also, it doesn't work, there's
+      # probably an issue with the `ADMINI~1` section
+      @system_temp_path = 'C:\\Windows\\Temp'
+      @external_copy_base = '/programdata'
```

Comment:
On Windows 2003, what I mean is that `C:\ProgramData` doesn't exist.  On that platform, `%ALLUSERSPROFILE%` resolves to `C:\Documents and Settings\All Users`

I think the issue in your example above might be with the escaping of `~`.  I don't believe that's necessary, and I think it's confusing SCP.